docs/quickstarts/ratify-on-azure.md

```diff
@@ -84,63 +256,49 @@ az role assignment create \
     Name                                                      RegistrationState
     --------------------------------------------------------    -------------------
     Microsoft.ContainerService/EnableWorkloadIdentityPreview    Registered
     ```
 
-1. Establish federated identity credential. On AZ CLI `${RATIFY_NAMESPACE}` is where you deploy Ratify:
+1. Update an existing AKS cluster with OIDC and workload identity enabled. You can skip this step if you have an AKS cluster with both OIDC and workload identity enabled.
+
+    ```shell
+    az aks update -g ${AKS_RG} -n ${AKS_NAME} --enable-oidc-issuer --enable-workload-identity
+    export AKS_OIDC_ISSUER="$(az aks show -n ${AKS_NAME} -g ${AKS_RG} --query "oidcIssuerProfile.issuerUrl" -otsv)"
+    ```
+
+1. Establish federated identity credential for namespace `${RATIFY_NAMESPACE}` where you deploy Ratify:
 
-    ```bash
+    ```shell
     az identity federated-credential create \
     --name ratify-federated-credential \
     --identity-name "${IDENTITY_NAME}" \
-    --resource-group "${GROUP_NAME}" \
+    --resource-group "${IDENTITY_RG}" \
     --issuer "${AKS_OIDC_ISSUER}" \
     --subject system:serviceaccount:"${RATIFY_NAMESPACE}":"ratify-admin"
     ```
 
-## Configure access policy for AKV
-
-1. Set the environmental variable for Azure Key Vault URI.
-
-    ```bash
-    export VAULT_URI=$(az keyvault show --name ${AKV_NAME} --resource-group ${GROUP_NAME} --query "properties.vaultUri" -otsv)
-    ```
-
-2. Ratify requires secret permissions to retrieve the public certificates for the entire certificate chain,
- please set private keys to Non-exportable at certificate creation time to avoid security risk. Learn more about non-exportable keys [here](https://learn.microsoft.com/en-us/azure/key-vault/certificates/how-to-export-certificate?tabs=azure-cli#exportable-and-non-exportable-keys)
-
-> Note: If you were unable to configure certificate policy, please consider specifying the public root certificate value inline using the [inline certificate provider](../reference/custom%20resources/certificate-stores.md#inline-certificate-provider) to reduce risk of exposing private key.
-
-Configure policy for user-assigned managed identity:
-
-```bash
-az keyvault set-policy --name ${AKV_NAME} \
---secret-permissions get \
---object-id ${IDENTITY_OBJECT_ID}
-```
-
-## Deploy Gatekeeper and Ratify on AKS
+## Install Gatekeeper and Ratify in AKS
 
-run `az aks show -g "${GROUP_NAME}" -n "${AKS_NAME}" --query addonProfiles.azurepolicy` to verify if the AKS cluster has azure policy addon enabled, learn more at [use azure policy](https://learn.microsoft.com/en-us/azure/aks/use-azure-policy)
+Run `az aks show -g "${AKS_RG}" -n "${AKS_NAME}" --query addonProfiles.azurepolicy` to verify if the AKS cluster has azure policy addon enabled, learn more at [use azure policy](https://learn.microsoft.com/en-us/azure/aks/use-azure-policy)
 
 ### When Azure Policy Addon is not enabled
```

Comment:
suggestion to add a link to "azure policy addon" for customer that is not aware of the AKS policy. also note that that Azure Policy is not required for this tutorial.

Reply:
@susanshi Is the link in the sentence good enough? Users can find the azure policy add-on in the link